db/migrate/20200813221545_add_post_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostRefToComments < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :comments, :post, null: false, foreign_key: true
+  end
+end
```

Comment:
#### You should create the same references to User.

Reply:
Thanks a lot explaining the issue

Reply:
Comments references on User class has been created. Thanks for the review.